src/Compilers/CSharp/Portable/Binder/Binder_Expressions.cs

```diff
@@ -5234,6 +5234,29 @@ private BoundExpression BindCollectionExpression(CollectionExpressionSyntax synt
 
             MessageID.IDS_FeatureCollectionExpressions.CheckFeatureAvailability(diagnostics, syntax, syntax.OpenBracketToken.GetLocation());
 
+            // Report any 'with(...)' elements not in the first element location.
+            for (int i = 1, n = syntax.Elements.Count; i < n; i++)
+            {
+                var element = syntax.Elements[i];
+                if (element is WithElementSyntax withElement)
+                {
+                    Error(diagnostics, ErrorCode.ERR_WithElementMustBeFirst, withElement.WithKeyword.GetLocation());
+                    break;
```

Comment:
Why are we breaking out of the loop?

Reply:
because we reported the error on the first vioaltor.  I didn't see any value in continuing to report on further violations :)

Reply:
taht said, binding behavior is entirely in chuck's court.  if he wants to change this, i have no issue with that.